packages/vega/test/index.html

```diff
@@ -22,7 +22,7 @@
       .vega-bind { line-height: 18px; margin-bottom: 2px; }
       .vega-bind-name { display: inline-block; width: 100px; }
       .vega-bind input[type="range"] { width: 400px; }
-      .vega-bind label { margin: 0 0.5em 0 2px; }
+      .vega-bind-radio label { margin: 0 0.5em 0 2px; }
```

Comment:
Why is the class here limited to radio? This change, in addition to dropping the use of the `label` tag for `range` elements, removes the formatting for the range value label. Was this intentional?

I think these changes may also break downstream deployments that expect specific element types and/or css selectors for their custom formatting. Does this necessitate a breaking (major version) change? (Note that I'm fine with dropping the option element id, I'm referring to the other changes.)

Can we keep the nice changes you've made here without breaking existing stylesheets?

Reply:
Ahh, good catch about the range label. I'll look into that.

The reason why radio is different is that radio is the only input type in vega bindings that has multiple inputs (dropdown, checkbox, range, and text are a single input per label).

I don't think we can not guarantee exactly the same selectors since I'm switching the type of HTML tag in some places. I'm all for reducing changes but this may be a case where we cannot make any changes without changing someone's layout (https://xkcd.com/1172/).

Reply:
`.vega-bind label` for example now selects elements that are nested differently.